doc/model.md

```diff
@@ -1,54 +1,56 @@
-# Istio Service Model
+# An Abstract Model of Services and Versions
 
-Istio's services model introduces the concept of a service version, a
-finer-grained service notion used to subdivide service instances by
-versions (`v1`, `v2`) or environment (`staging`, `prod`). Istio routing
-rules can refer to the service versions, to provide additional control over
-traffic between services.
+The model described below assumes a microservice-based application owned by
+a single tenant.
 
-The Istio model of services in the application is independent of the
-underlying platform (Kubernetes, Mesos, CloudFoundry, etc.). Platform
-specific adapters are responsible for populating the internal model
-representation with various fields, from the metadata found in the
-platform. The following sections describe various aspects of the abstract
-service model used by Istio.
+**Istio-managed service:** Modern cloud applications are created by
+composing together independent sets of services. Individual services are
+referenced by their fully-qualified domain name (FQDN) and one or more
+ports where the service is listening for connections.
 
-## Service
+**Instances:** Each service has one or more instances, i.e., actual
+manifestations of the service. Instances represent entities such as
+containers, pods (kubernetes/mesos), VMs, etc. 
```

Comment:
Yikes. This is like the cloud foundry model. The model described here is what the manager uses internally. I think we can kill this notion of instances for the moment, because you are now talking about multi-tenant services. I prefixed the entire document with a statement saying that this applies only to a single tenant's deployment. In a multi-tenant scope, one would have to add the tenant ID, and if logical instance, add the physical network endpoint. The discussion then becomes even more deeper and I need to think more on how to water it down. or use a diagram.